src/manifest/Application.ts

```diff
@@ -28,6 +29,8 @@ export class Application {
    */
   public api?: ManifestApi;
```

Comment:
https://github.com/getoutreach/extensibility-sdk/blob/main/src/manifest/ManifestApi.ts#L49
this should be removed